jans-client-api/uma-rs-resteasy/src/main/java/io/jans/ca/rs/protect/resteasy/ResourceRegistrar.java

```diff
@@ -112,7 +112,7 @@ private void register(RsResource rsResource) {
 
         } catch (Exception ex) {
             LOG.error(ex.getMessage(), ex);
-            throw ex;
+            throw (RuntimeException) ex;
```

Comment:
It is problem, not every exception is RuntimeException. Why do we have to change this line at all ?

Reply:
One sonar issue was https://rules.sonarsource.com/java/type/Code%20Smell/RSPEC-112
due to RuntimeException throw, I prefered to supress the sonar warning to avoid more changes.